### PR TITLE
Added intro voice interaction

### PR DESCRIPTION
**Pull request by saurabh28march**
It gives a brief introduction of various steps to be performed by the user.

### CODE REVIEW

**Comment by saurabh28march** [comment]
@maze-runnar  @SARTHAK1SINGH  review it. Then we move on to next feature.

File: templates/result.html [hunk 1]
```diff
@@ -225,15 +229,12 @@ <h4 style="color:red">Your medicine/drug is not appropriate for you condition.</
 <!-- very neg / very pos section -->
     <div class="suitable">
         <h4 style="color: rgb(145, 9, 9);">Most Negative</h4>  
-        <h4>{{most_negative_review}}</h4>
+        <h4 style="color: rgb(145, 9, 9);>{{most_negative_review}}</h4>
     </div>
     <div class="suitable">
```
Comment (maze-runnar) [inline templates/result.html:234]: forgot close inverted commas

File: templates/result.html [hunk 2]
```diff
@@ -225,15 +229,12 @@ <h4 style="color:red">Your medicine/drug is not appropriate for you condition.</
...
         <h4 style="color: rgb(6, 94, 3);">Most Positive</h4>  
-        <h4>{{most_positive_review}}</h4>
+        <h4 style="color: rgb(6, 94, 3);>{{most_positive_review}}</h4>
```
Comment (maze-runnar) [inline templates/result.html:236]: forgot close inverted commas in style = ""

Comment (maze-runnar) [inline templates/result.html:236]:
```suggestion
        <h4 style="color: rgb(6, 94, 3)";>{{most_positive_review}}</h4>
```

File: templates/result.html [hunk 3]
```diff
@@ -225,15 +229,12 @@ <h4 style="color:red">Your medicine/drug is not appropriate for you condition.</
 <!-- very neg / very pos section -->
     <div class="suitable">
         <h4 style="color: rgb(145, 9, 9);">Most Negative</h4>  
-        <h4>{{most_negative_review}}</h4>
+        <h4 style="color: rgb(145, 9, 9);>{{most_negative_review}}</h4>
```
Comment (maze-runnar) [inline templates/result.html:232]:
```suggestion
        <h4 style="color: rgb(145, 9, 9)";>{{most_negative_review}}</h4>
```

**Comment by saurabh28march** [comment]
@maze-runnar  done!!

**Comment by maze-runnar** [comment]
@SARTHAK1SINGH merge

**Pull request merged**